### DISCUSSION TITLE
What minimum permissions needed to allow revisions in the Directus app?

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#12016] [Archive]
We have a custom `Manager` role with the `App Access Minimum` permissions.

But we want to add the permissions to see the Revisions app option:

<img width="299" alt="Screenshot 2022-03-07 at 18 43 14" src="https://user-images.githubusercontent.com/1007051/157088580-7a318c89-54ab-445a-8720-4df7293851f8.png">

We set this, but seems like it's not enough:

<img width="826" alt="Screenshot 2022-03-07 at 18 44 04" src="https://user-images.githubusercontent.com/1007051/157088792-536388e5-c74f-47c1-8146-1e071ef36717.png">

**Comment by rijkvanzanten** [comment]
I believe that section shows every row from directus_activity and then relates in directus_revisions. Are the permissions to directus_activity permissive as well?

**Reply by abdonrd** [reply]
Thanks Rijk!

Seems like if I also add the all access read permissions to the `directus_activity` it's working:

<img width="820" alt="Screenshot 2022-03-07 at 18 57 36" src="https://user-images.githubusercontent.com/1007051/157090880-f9d3f9b7-9280-452c-b009-c7bbfa86bde9.png">

<img width="826" alt="Screenshot 2022-03-07 at 18 44 04" src="https://user-images.githubusercontent.com/1007051/157088792-536388e5-c74f-47c1-8146-1e071ef36717.png">
